Add missing options for RN (#4828)
getsentry/sentry-docs: src/platforms/common/configuration/options.mdx

</ConfigKey>

<ConfigKey name="enableNativeCrashHandling" supported={["react-native", "javascript.capacitor"]}>

> Set this boolean to `false` to disable the hard crash handling from the native layer. Doing so means it won't capture events for hard crashes on Android and iOS if the error was caused by native code.

Reviewer: what is "it" in the second sentence? Let's spell that out just to be super clear. 

```suggestion
Set this boolean to `false` to disable hard crash handling from the native layer. Doing so means it won't capture events for hard crashes on Android and iOS if the error was caused by native code.
```

Author: `it` means the Sentry RN SDK.
https://github.com/getsentry/sentry-docs/pull/4828/commits/99ed44716e6ab09a1c3adc5988e3d5072071e3b8

</ConfigKey>

<ConfigKey name="enableNativeNagger" supported={["react-native", "javascript.capacitor"]}>

> Set this boolean to `false` to disable the native nagger alert be shown.

Reviewer: ```suggestion
Set this boolean to `false` to disable the native nagger alert being shown.
```

</ConfigKey>

<ConfigKey name="enableAutoSessionTracking">

Set this boolean to `false` to disable the [release health](/product/releases/health/) feature.

</ConfigKey>

<ConfigKey name="sessionTrackingIntervalMillis">

> Set this to change the default interval to end a session (release health) if the App goes to the background, Default is 30000.

Reviewer: ```suggestion
Set this to change the default interval to end a session (release health) if the app goes to the background. Default is 30,000.
```

</ConfigKey>

<ConfigKey name="enableNdkScopeSync">

Set this boolean to `false` to disable the scope sync from Java to NDK on Android.

</ConfigKey>

<ConfigKey name="attachThreads">

> Set this boolean to `true` to automatically attach all the threads to all logged events on Android.

Reviewer: ```suggestion
Set this boolean to `true` to automatically attach all threads to all logged events on Android.
```

</ConfigKey>

<ConfigKey name="enableAutoPerformanceTracking" supported={["react-native"]}>

> Set this boolean to `false` to disable the auto [performance monitoring](product/performance/) tracking.

Reviewer: ```suggestion
Set this boolean to `false` to disable auto [performance monitoring](/product/performance/) tracking.
```

</ConfigKey>

<ConfigKey name="enableOutOfMemoryTracking" supported={["react-native"]}>

> Set this boolean to `false` to disable the [out of memory](platforms/apple/guides/ios/configuration/out-of-memory/) tracking on iOS.

Reviewer: ```suggestion
Set this boolean to `false` to disable [out of memory](/platforms/apple/guides/ios/configuration/out-of-memory/) tracking on iOS.
```

</ConfigKey>

<ConfigKey name="onReady" supported={["react-native"]}>

> Set this callback that is called after the Sentry React Native SDK initialized its Native SDKs (Android and iOS).

Reviewer: ```suggestion
Set this callback, which is called after the Sentry React Native SDK initializes its Native SDKs (Android and iOS).
```
